app/assets/stylesheets/itp.css

```diff
@@ -0,0 +1,5 @@
+#RequestStorageAccess,
+#CookiePartitionPrompt,
+#TopLevelInteractionContent {
+  display: none;
+}
```

Comment:
why do these need to be in a separate CSS file? actually feels like we might be able to inline these to cut down on HTTP roundtrips.